eras/conway/impl/test/Test/Cardano/Ledger/Conway/GoldenSpec.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Conway.GoldenSpec (spec) where
+
+import Cardano.Ledger.Conway
+import Test.Cardano.Ledger.Common
+import Test.Cardano.Ledger.Core.JSON (goldenPParamsSpec)
+
+spec :: Spec
+spec =
+  describe "Golden" $ do
+    goldenPParamsSpec @ConwayEra ["golden/pparams-full.json"]
```

Comment:
You need to convert this relative path to the full path, like we do with CDDL, otherwise it will break some setups, especially nix.
```suggestion
    goldenPParamsSpec @ConwayEra =<< mapM getDataFileName ["golden/pparams-full.json"]
```
This comment applies to all calls to `goldenPParamsSpec`

Reply:
Had to wrap `getDatafileName` in `runIO`: 

```
goldenJsonPParamsSpec @ShelleyEra =<< runIO (getDataFileName "golden/pparams.json")
```

Reply:
Hmm. Not good! Convert from `goldenPParamsSpec :: Spec` to `goldenPParamsSpec :: SpecWith [FilePath]`

Reply:
Makes sense, thanks!